python/ecl/grid/ecl_grid.py

```diff
@@ -1316,6 +1348,12 @@ def export_position(self, index_frame):
         return data
 
     def export_corners(self, index_frame):
+        """Exports cell corner position coordinates to a numpy vector (matrix). 
+     
+        Index_fram must be a pandas dataframe with the same structure 
+        as obtained from export_index.  
+        Columns 3m, 3m+1 and 3m+2 are x, y, and z coordintates of corner m. 
```

Comment:
In total there are eight 8 corners; and they are numbered as this: https://github.com/Statoil/libecl/blob/master/lib/ecl/ecl_grid.cpp#L464